docs/learn/learn-transaction-fees.md

```diff
@@ -24,23 +24,48 @@ designed the Polkadot fee system with the following objectives:
 
 Fees on the Polkadot Relay Chain are calculated based on three parameters:
 
-- A per-byte fee (also known as the "length fee").
-- A weight fee.
-- A tip (optional).
+- A Weight fee
+  - Base weight
+  - Call(s) weight
+- A Length fee
+- A Tip (optional).
 
-The length fee is the product of a constant per-byte fee and the size of the transaction in bytes.
+Weights are a fixed set of numbers used in Substrate-based chains to manage the time it 
+takes to validate a block. Each transaction has a base weight that accounts for the overhead 
+of inclusion (e.g. signature verification) and a dispatch weight that 
+accounts for the time to execute the transaction. All weights, even the base weight, 
+are a measure of time to execute on some standard hardware.
 
-Weights are a fixed number designed to manage the time it takes to validate a block. Each
-transaction has a base weight that accounts for the overhead of inclusion (e.g. signature
-verification) as well as a dispatch weight that accounts for the time to execute the transaction.
-The total weight is multiplied by a per-weight fee to calculate the transaction's weight fee.
+The runtime 
+[converts weight units to balance units](https://docs.substrate.io/how-to-guides/v3/weights/calculate-fees/) 
+as part of the fee calculation.
 
-Tips are an optional transaction fee that users can add to give a transaction higher priority.
+The weight fee is the sum of the base weight and the sum of the total weight consumed by 
+call(s). 
 
-Together, these three fees constitute the inclusion fee. This fee is deducted from the sender's
-account prior to transaction execution. A portion of the fee will go to the block producer and the
-remainder will go to the [Treasury](learn-treasury.md). At Polkadot's genesis, this is set to 20%
-and 80%, respectively.
+> A transaction can include several calls. For instance, a `batch` can contain `bond` and `nominate`, 
+> and the weight would be one base weight and then the sum of the weights for `bond` and `nominate`.
+
+To learn more about the motivation of a weight fee, check out this 
+[Substrate doc](https://docs.substrate.io/v3/concepts/weight/) on weights.
+The base fee is the minimum a user pays for a transaction. 
```

Comment:
I would actually remove this sentence. It's only true in theory (if a call had 0 weight and 0 length), and a bit confusing because readers might think that they can actually get fees down to this level.

In practice, the inclusion fee is the minimum fee to get a transaction included.

Reply:
Right makes sense, updated:)